Update cookie expiration time (#5077)
cloudflare/cloudflare-docs: content/waiting-room/reference/waiting-room-cookie.md

## Cookie expiration time

> - While a visitor stays in a waiting room, the `__cfwaitingroom` cookie is set to expire in 5 minutes.

Reviewer: ```suggestion
- While a visitor stays in a waiting room, the `__cfwaitingroom` cookie will not expire.
- If a user closes their tab or leaves the waiting room, their cookie will expire after five minutes.
```

> - While a visitor stays in a waiting room, `__cfwaitingroom` cookie expiration is always set to five minutes, but renews every 20 seconds automatically as long as the visitor does not close their tab or leaves your application.

Reviewer: ```suggestion
- While a visitor stays in a waiting room, `__cfwaitingroom` cookie expiration is always set to five minutes, but renews every 20 seconds automatically as long as the visitor does not close the tab or leaves your application.
```

> - While a visitor stays in a waiting room, `__cfwaitingroom` cookie expiration is always set to five minutes, but renews every 20 seconds automatically as long as the visitor does not close the their tab or leave your application.

Reviewer: ```suggestion
- While a visitor stays in a waiting room, `__cfwaitingroom` cookie expiration is always set to five minutes, but renews every 20 seconds automatically as long as the visitor does not close their tab or leaves your application.
```

## Cookie expiration time

> - While a visitor stays in a waiting room, the `__cfwaitingroom` cookie will not expire.

Reviewer: ```suggestion
- While a visitor stays in a waiting room, `__cfwaitingroom` cookie expiration is always set to five minutes, but renews every 20 seconds automatically as long as the visitor does not close the their tab or leave your application.
```

> - If a user closes their tab or leaves the waiting room, their cookie will expire after five minutes.

Reviewer: ```suggestion
```
